tensorflow/examples/tutorials/mnist/fully_connected_feed.py

```diff
@@ -151,7 +151,7 @@ def run_training():
     sess = tf.Session()
 
     # Instantiate a SummaryWriter to output summaries and the Graph.
-    summary_writer = tf.train.SummaryWriter(FLAGS.train_dir, sess.graph)
+    summary_writer = tf.train.SummaryWriter(FLAGS.data_dir, sess.graph)
```

Comment:
it doesn't matter, I think, it basically write the graph to the directory, it has nothing to do with whether it's training data or not